S7.Net/PLC.cs

```diff
@@ -1113,8 +1257,8 @@ private ErrorCode WriteBytesWithASingleRequest(DataType dataType, int db, int st
 
                 _mSocket.Send(package.array, package.array.Length, SocketFlags.None);
 
-                int numReceived = _mSocket.Receive(bReceive, 512, SocketFlags.None);
-                if (bReceive[21] != 0xff)
+                int numReceived = ReciveCOTP_TSDU(bReceive);
+                if (bReceive[14] != 0xff)
```

Comment:
why did this change from 21 to 14 ?

Reply:
The reason is that cotp_tsdu.data only contains the S7comm data. Both headers for tpkt and cotp are removed.